### ISSUE TITLE
conda.exe not found during install

### ISSUE THREAD

**Issue by latentcode** [conda-forge/miniforge#440]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

**Environment**: Raspberry Pi 4 B 8GB
**uname -a**: Linux raspberrypi 6.1.19-v8+ #1637 SMP PREEMPT Tue Mar 14 11:11:47 GMT 2023 aarch64 GNU/Linux

I just set up the Raspberry Pi described above and I'm trying to install Miniforge3 using **Miniforge3-Linux-aarch64.sh**, but I run into this problem:

> Miniforge3 will now be installed into this location:
> /home/me/miniforge3
> 
>   - Press ENTER to confirm the location
>   - Press CTRL-C to abort the installation
>   - Or specify a different location below
> 
> [/home/me/miniforge3] >>> 
> PREFIX=/home/me/miniforge3
> Unpacking payload ...
> Miniforge3-Linux-aarch64.sh: line 311: /home/me/miniforge3/conda.exe: No such file or directory

Contrary to the message, **conda.exe** appears to exist:

> me@raspberrypi:~/Downloads $ ls -la /home/me/miniforge3/conda.exe
> -rwxr-xr-x 1 me me 17866928 Apr  4 18:17 /home/me/miniforge3/conda.exe

Interestingly, it doesn't work when I do it myself either:

> me@raspberrypi:~/Downloads $ /home/me/miniforge3/conda
> bash: /home/me/miniforge3/conda: No such file or directory

I considered the possibility that it's a problem w/ my setup rather than conda, but i don't have the problem w/ other executables:

> me@raspberrypi:~/Downloads $ pyenv --version
> pyenv 2.3.16

### Installed packages

```shell
N/A - install issue
```

### Environment info

```shell
N/A - install issue
```

**Comment by cboulay** [comment]
I'm trying to help someone with this exact problem. No issues on my pi 4b (`Linux raspberrypi 5.15.84-v8+ #1613 SMP PREEMPT Thu Jan 5 12:03:08 GMT 2023 aarch64 GNU/Linux`), but they encounter the same issue as described above (`Linux raspberrypi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux`).

**Comment by hmaarrfk** [comment]
kernel issue?

**Comment by shsrams** [comment]
I had a similar problem where I was trying to run the Miniforge3-Linux-aarch64.sh and was seeing "conda.exe not found" error. Eventually realized that I had Raspberry Pi OS (32-bit) installed and that was the culprit. Once I switched to Raspberry Pi OS (64-bit) the installation proceeded smoothly.

**Comment by jbuusao** [comment]
I've got the same issue (conda.exe not found), despite running 64bits:
Linux rpi4jp 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux

**Comment by cyclemaxwell** [comment]
@jbuusao, I just encountered this today. ThoughtI was on 64 bit, but actually this post explains how one can be on 32 bit and be mislead:
https://lindevs.com/check-whether-raspberry-pi-os-is-32-bit-or-64-bit

Just fresh installed 64 bit and installed Mambaforge after failing every variation of Mamba/Mini/\[forge|conda\] on what I had incorrectly identified as 64 bit earlier.

**Comment by jbuusao** [comment]
@cyclemaxwell You were correct, thank you for the tip. Indeed it all works now!

**Comment by latentcode** [comment]
Closing this bug as resolved.

I can't explain the original problem. Given that **uname -a** reported **aarch64**, I was confident that I was running a 64 bit version: 
Linux raspberrypi 6.1.19-v8+ #1637 SMP PREEMPT Tue Mar 14 11:11:47 GMT 2023 aarch64 GNU/Linux

I've re-installed Linux a few times for various reasons since opening this ticket. I tried **Miniforge3-Linux-aarch64.sh** again today and it installed w/o a hitch. the current **uname -a** output indicates a minor Linux version revision:
Linux raspberrypi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux
